frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.container.jsx

```diff
@@ -1,129 +1,62 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import qs from 'qs';
-import pickBy from 'lodash/pickBy';
 import sortBy from 'lodash/sortBy';
-import DashboardWidget from 'react-components/dashboard-element/dashboard-widget/dashboard-widget.component';
+import DashboardWidgetComponent from 'react-components/dashboard-element/dashboard-widget/dashboard-widget.component';
 import DashboardWidgetTooltip from 'react-components/dashboard-element/dashboard-widget/dashboard-widget-tooltip';
-import Widget from 'react-components/widgets/widget.component';
-import CHART_CONFIG from 'react-components/dashboard-element/dashboard-widget/dashboard-widget-config';
+import { getConfig } from 'react-components/dashboard-element/dashboard-widget/dashboard-widget.selectors';
+
+const mapStateToProps = (state, ownProps) => {
+  const selectorState = {
```

Comment:
you don't need to pass it like this, this can cause confusion on the selector function, one could think the redux state has a different structure. If you need to pass own props, you can pass them as arguments, and consume them from the selector function.